redhat-developer/rhdh-operator
# Replace default config with raw, do not merge

## Description

Replace default config with raw, do not merge

## Which issue(s) does this PR fix or relate to

https://issues.redhat.com/browse/RHIDP-3806

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
For example:

- Add label1 to default config service.yaml
- Add label2 to raw service.yaml
- Create CR with raw config
Make sure only label2 present
